parm/config/config.aeroanl

```diff
@@ -5,13 +5,19 @@
 
 echo "BEGIN: config.aeroanl"
 
-export OBS_YAML_DIR=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/config/
-export OBS_LIST=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/lists/aero_prototype.yaml
-export AEROVARYAML=$HOMEgfs/sorc/gdas.cd/parm/aero/variational/3dvar_dripcg.yaml
-export BERROR_YAML=$HOMEgfs/sorc/gdas.cd/parm/aero/berror/static_bump.yaml
-export FV3JEDI_FIX=$HOMEgfs/fix/gdas
+export CASE_ANL=${CASE}
```

Comment:
Is there a reason for this `CASE_ANL`?  Can, `$CASE` not be used where `$CASE_ANL` is used.

Reply:
This is in case the background and analysis are on different resolutions. This may be different than `CASE_ENKF` which is used for the ATM.